### PR TITLE
Validating that a note isn't blank

### LINKED ISSUE #11: It's possible to send a notable message without content
Simply entering `/notable` will accept the empty message and broadcast it.

The command should validate input and at least require the message to be non-empty.

### PR DESCRIPTION
**Pull request by jasondew**
Fixes #11 

@zmoazeni Would you be so kind as to take a look?

### CODE REVIEW

File: notable.go [hunk 1]
```diff
@@ -17,7 +18,11 @@ func targetRoom() string {
 	return "general"
 }
 
-func Record(authorID string, category string, text string, slackToken string) {
+func Record(authorID string, category string, text string, slackToken string) error {
+	if len(strings.Trim(text, " ")) == 0 {
```
Comment (zmoazeni) [inline notable.go]: [nb] FWIW this `strings.Trim(text, " ")` might be good to do regardless at the beginning of the method. Would we want any notable announcements to go out with leading or trailing spaces?

**Comment by zmoazeni** [comment]
+1

File: notable.go (hunk 1 above)
Comment (jasondew) [inline notable.go]: Ah, actually we're already `TrimSpace`ing a few lines down so we're good.

**Comment by jasondew** [comment]
Thanks @zmoazeni!

**Pull request merged**